doc/changes/changes_4.0.0.md

```diff
@@ -22,4 +22,5 @@ The old variant is intentionally not supported anymore to tighten security.
 
 ## Plugin Updates
 
+* Added `project-keeper-maven-plugin:0.4.0`
 * Updated `jacoco-maven-plugin:0.8.5` to `0.8.6`
```

Comment:
You also added `udf-debugging-java` and the `maven-dependency-plugin`

Reply:
Fixed.